### PR TITLE
BAU - DMS Schema update

### PR DESCRIPTION
**Pull request by PauloPortugal**
**Elements being added to the Declaration Element:**
```
Communication  - (Declaration/Consignment/Carrier/Communication) (42A/28A/18A/25A) optional
```

**Elements being removed from the Declaration Element:**
```
Manufacturer - (Declaration/GoodsShipment/GovernmentAgencyGoodsItem/Manufacturer) (42A/67A/68A/86A) optional
RefundRecipientParty - (Declaration/GoodsShipment/GovernmentAgencyGoodsItem/RefundRecipientParty) (42A/67A/68A/59B) optional
```

### CODE REVIEW

File: src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd [hunk 1]
```diff
@@ -1,103 +1,111 @@
...
+              <XMLTag>AgencyAssignedCustomizationVersionCode</XMLTag>
+              <WCOName>Agency Assigned Customization Version</WCOName>
+              <UniquePositionID>9017</UniquePositionID>
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:choice>
+          <xs:group ref="Declaration"/>
+          <xs:group ref="Response" maxOccurs="unbounded"/>
+        </xs:choice>
```
Comment (PatrykRudnicki) [inline src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd:97]: This part is added

File: src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd [hunk 2]
```diff
@@ -1,103 +1,111 @@
...
+  <xs:group name="Declaration">
+    <xs:choice>
+      <xs:any namespace="urn:wco:datamodel:WCO:DEC-DMS:2" processContents="strict"/>
+    </xs:choice>
+  </xs:group>
+  <xs:group name="Response">
+    <xs:sequence>
+      <xs:any namespace="urn:wco:datamodel:WCO:RES-DMS:2" processContents="strict"/>
+    </xs:sequence>
+  </xs:group>
```
Comment (PatrykRudnicki) [inline src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd:110]: and this

File: src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd (hunk 1 above)
Comment (PatrykRudnicki) [reply src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd:97]: I mean only response

Comment (PauloPortugal) [reply src/main/resources/wco/declaration/DocumentMetaData_2_DMS.xsd:97]: Yes!!! Well spotted!

**Review by PatrykRudnicki — APPROVED** [review]

**Comment by PauloPortugal** [comment]
This was just to inspect the changes. The dec-api team have already made the changes on their repo and we have https://github.com/hmrc/wco-dec/pull/25/files covering it

**Pull request closed without merging**